coala_quickstart/Strings.py

```diff
@@ -33,3 +33,9 @@
              "all its contents but exclude your .git directory and all "
              ".o files. To do this, simply give `src/` for the first "
              "question and `.git/**,**/*.o` for the second question.")
+
+bear_help = ("A coala bear is a plugin that contains the actual algorithm. "
+             "It have its own native code or use an external linter. It "
+             "may be language specific or language independent. This "
+             "makes coala completely modularized and extensible. "
+             "Currently there are 78 bears for 36 different languages.")
```

Comment:
I think we should also add a link to the bear docs README, saying "You can see the full list here" or something like that